egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -392,6 +392,7 @@ if ! "${skip_data_prep}"; then
                 utils/fix_data_dir.sh "${dumpdir}/mfcc/${dset}"
 
                 # 3. Compute VAD decision
+                _nj=$(min "${nj}" "$(<${dumpdir}/mfcc/${dset}/spk2utt wc -l)")
```

Comment:
Note that I modify the nj here to align the jobs with the number of speakers instead of number utterances, as following jobs generally depends on speaker number.

Reply:
Thanks!